website/pages/en/versions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary');
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
+
+const CWD = process.cwd();
+
+const siteConfig = require(CWD + '/siteConfig.js');
+const versions = require(CWD + '/versions.json');
+const githubReleaseUrl = version => `https://github.com/wix/detox/releases/tag/${version}`;
+const documentationTarget = "Introduction.GettingStarted.html"
+
+function VersionLinks({version, isLatest = false}) {
+  const expandedVersion = version.replace('.X', '.0.0')
+  const isMaster = version === "master";
```

Comment:
Not sure I get what the final result will be. Will site show exact version (i.e. 7.3.5) or 7.0.0 throughout links ?

Reply:
Video shows 7.X, 6.X and master. what am I missing ?

Reply:
Video is old, but you can run it locally. Just do `cd website/ && npm install && npm run gatherDocs && npm start` and you can see the site